doc/manual/de/install/docker.rst

```diff
@@ -40,25 +40,34 @@ Umgebungsvariablen (environment)
 Der Container wird über Umgebungsvariablen grundsätzlich konfiguriert.
 Zur Verfügung stehen:
 
-+---------------------+------------------------------+-----------------------------------------------------------------+
-|Parameter            |Default                       |Beschreibung                                                     |
-+=====================+==============================+=================================================================+
-|KNX_INTERFACE        |``iptn:172.17.0.1:3700``      |knxd-URL für die KNX Schnittstelle. Wenn leer, so wird der       |
-|                     |                              |``knxd`` nicht geladen.                                          |
-+---------------------+------------------------------+-----------------------------------------------------------------+
-|KNX_PA               |``1.1.238``                   |Physikalische Schnittstelle (PA) für den ``knxd``.               |
-+---------------------+------------------------------+-----------------------------------------------------------------+
-|KNXD_PARAMETERS      |``-u -d/var/log/eibd.log -c`` |Weitere Aufruf-Parameter für den ``knxd``.                       |
-+---------------------+------------------------------+-----------------------------------------------------------------+
-|CGI_URL_PATH         |``/cgi-bin/``                 |URL-Prefix um die ``cgi-bin`` Ressourcen zu lokalisieren.        |
-+---------------------+------------------------------+-----------------------------------------------------------------+
-|BACKEND_PROXY_SOURCE |                              |Proxy-Pfade beginnen mit diesem Wert, z.B. ``/rest`` für das     |
-|                     |                              |openHAB Backend.                                                 |
-+---------------------+------------------------------+-----------------------------------------------------------------+
-|BACKEND_PROXY_TARGET |                              |Ziel-URL für die Proxy-Anfragen an ``BACKEND_PROXY_SOURCE``,     |
-|                     |                              |z.B. ``http://<openhab-server-ip-address>:8080/rest`` für das    |
-|                     |                              |openHAB Backend.                                                 |
-+---------------------+------------------------------+-----------------------------------------------------------------+
+.. list-table::
+   :header-rows: 1
+
+   * - Parameter
+     - Default
+     - Beschreibung
+   * - KNX_INTERFACE
+     - 
```

Comment:
``iptn:172.17.0.1:3700`` is missing.

Reply:
I removed the default value on purpose. The knxd in the container is only started when that value is set, it should not start with a random interface value, that most likely does not work because the user needs to set it for his setup to another value.